### PR TITLE
Updates to scripting documentation: Impute and typos

### PR DESCRIPTION
**Pull request by ajdapretnar**
##### Issue
Scripting documentation missing reference on imputation.

##### Description of changes
A sorry attempt at updating scripting documentation. Some typos fixed. Imputation added.

##### Includes
- [X] Code changes
- [ ] Tests
- [X] Documentation

### CODE REVIEW

File: doc/data-mining-library/source/reference/classification.rst [hunk 1]
```diff
@@ -85,11 +85,6 @@ For data sets that include continuous attributes,
 .. index:: multilayer perceptron
    pair: classification; multilayer perceptron
 
-Multilayer perceptron (feed-forward neural network)
```
Comment (lanzagar) [inline doc/data-mining-library/source/reference/classification.rst]: The index above the removed section should be removed too. It is probably not used anywhere, but it wouldn't hurt to double check.

File: doc/data-mining-library/source/reference/code/imputation-default.py [hunk 2]
```diff
@@ -0,0 +1,7 @@
+from Orange.data import Table
+from Orange.preprocess import Impute
+
+data = Table("heart-disease.tab")
+imputer = Impute()
+
+impute_heart = imputer(data)
```
Comment (lanzagar) [inline doc/data-mining-library/source/reference/code/imputation-default.py]: missing newline at EOF
(a properly configured text editor should take care of these automatically)

File: doc/data-mining-library/source/reference/classification.rst (hunk 1 above)
Comment (ajdapretnar) [inline doc/data-mining-library/source/reference/classification.rst]: Agree. Overlooked.

File: doc/data-mining-library/source/reference/code/imputation-default.py (hunk 2 above)
Comment (ajdapretnar) [inline doc/data-mining-library/source/reference/code/imputation-default.py]: Will fix and reconfigure editor.

**Comment by ajdapretnar** [comment]
@lanzagar Should be fixed now.

**Pull request merged**